README.md

```diff
@@ -1,135 +1,221 @@
-However, a simple rule is any time you create or update a **category**, **product**, **customer** or **order**, call the relevant [Backend endpoint](#backend-endpoints) with the required data.
-
-:warning: Keep in mind that:
-* if you try to make any [Frontend event](#frontend-events) for not existing resource in Metrilo, your request won't be processed.
-* most of the times the data you send will override any existing data. For example, if you make a call about product *A* with categories *C1* and *C2* and then you make another call only with category *C3*, then the first two categories will be removed from the product. However, sometimes the data will be merged(e.g. product's options). If the data is merged, it will be explicitly mentioned in the OpenAPI documentation.
-
-### Backend endpoints
-
-* /category
-* /product
-* /customer
-* /order
-
-More detailed information about what data they require you can find in the OpenAPI documentation mentioned above.
-
-### Example of when to use these endpoints:
-| when                                                                                        | you have to call ... with the required data        |
-| :------------------------------------------------------------------------------------------ | :------------------------------------------------- |
-| a customer signs up for first time                                                          | /customer to create it                             |
-| a customer profile is updated                                                               | /customer to update it                             |
-| a new category is added to a product                                                        | /product to update its categories                  |
-| a new product is created                                                                    | /product to create it                              |
-| a new order is made                                                                         | /order to create it                                |
-| an order's status or details are changed                                                    | /order to update it                                |
-| a customer doesn't sign up, but makes an order just by filling in his email(guest customer) | /customer and then /order to create both resources |
-
-
-<br/><br/>
-## Responses from the endpoints
-The following responses are valid for all of the endpoints.
-
-| Code | Message |  
-| --- | --- |  
-| 204 | Success response. |  
-| 400 | Bad request from client. |  
-| 401 | Project token is invalid. |  
-| 402 | Project requires payment. |  
-| 403 | The IP that the request came from is ignored. |  
-| 500 | Error from server while processing request. |  
-| 502 | Error from server accepting request. |  
+**Note: This guide is intended to be used by developers. If you're not a developer - please find one and come back later :)**
```

Comment:
I think this `Note` should be a :warning: emoji.

Reply:
Will do